### ISSUE TITLE
error message piles up in ui "notebook file has changed on disk.."

### ISSUE THREAD

**Issue by steverweber** [jupyter/notebook#1640]
Because of an issue with how I have to mount the file system I get this error message.

```
The notebook file has changed on disk since the last time we opened or saved it. Do you want to overwrite the file on disk with the version open here, or load the version on disk (reload the page) ?
```

I's fine that i get the error message however every 60 or so seconds one will spawn overtop of one that is already displayed... this continues forever...

When I return to my office sometimes I have to close more than 60 of them.

Please have the error check if one is already displayed before creating a new one ontop of the old one.

thanks

**Comment by Carreau** [comment]
Thanks, we are aware of this issues, though it need a bit of refactoring to decide wether 2 dialog are the same, or should not overlay.

**Comment by takluyver** [comment]
@Carreau is this something we can get fixed for 5.0 (which I'm trying to make soon)? If not, please bump it to 5.1.

**Comment by minrk** [comment]
I think this one is easy: Don't draw another dialog if one is already there. I can have a fix for that shortly.

 I think we have other Issues tracking when this dialog occurs spuriously, which I think we can't fix for 5.0.

**Comment by minrk** [comment]
#2098 should do it.
